include/gnc_constants.hpp

```diff
@@ -0,0 +1,81 @@
+//
+// include/gnc_constants.hpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+/* - NOTICE -
+ * 
+ * Talk to Kyle Krol and/or Nathan Zimmerberg before editing this file.
+ *
+ * Values in this file should be seldomly edited and have hidden
+ * interdependancies. The interdependancies and explicit calculation of the
+ * below parameters is specified in MATLAB/config.m. */
+
+#ifndef PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+#define PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+
+#include "lin.hpp"
+
+namespace gnc {
+namespace constant {
+
+/** Good old pi. */
+constexpr static double pi = 3.141592653589793;
+
+/** Two pi. */
+constexpr static double two_pi = 2.0 * pi;
+
+/** Initial GPS week number that serves as the 'PAN epoch'. The 'PAN epoch'
+ *  aligns with a GPS week so no additional offset is required. */
+constexpr static unsigned short init_gps_week_number = 2045;
```

Comment:
@tanishqaggarwal It wouldn't let me continue the other conversation for some reason. To answer your earlier question however about adding in a nanosecond count for the 'PAN epoch', I'm pretty sure we're setting an epoch at the very start of a GPS week so the initial nanosecond count is zero.

@nhz2 could you confirm.

Reply:
I agree it should be zero, but for the sake of having a named constant, I think it would be a good idea to have the TOW and nanoseconds explicitly defined?

Reply:
Alright yeah that's fair just in case we do something goofy down the road. Will add it in!